### ISSUE TITLE
[Accessbility]A11y_ Java Tooling for IntelliJ/Eclipse – New Feature_Application Graph_Keyboard: User cannot be able to access “Driver” and ”Jobs” with the help of keyboard

### ISSUE THREAD

**Issue by InduPriya1805** [microsoft/azure-tools-for-java#3597]
GitHub Tags: #A11y_JavaToolingforIntelliJ/Eclipse–NewFeature; #A11yMAS; #A11yTCS; #C+E-H; #DesktopApp; #Win10; #JavatoolingforIntelliJ/Eclipse; #Keyboard; #MAS2.1.1; 

[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier

**Environment Details:**
Download and Install IntelliJ IDEA from https://www.jetbrains.com/idea/download/#section=windows
IntelliJ IDEA 2019.2.1
Build #IU-192.6262.58

**OS Details:**
OS: Windows (Version 1903, Build: 18362.113)

**MAS Violated**: 2.1.1

**Repro Steps:**
1)Launch IntelliJ IDEA
2)Tab till Create New Project button and press enter. Navigate to Azure Spark/HDInsight-->Spark Project Scala and press Next button and fill all the details to create a new project
3)Tab till Azure Spark/HDInsight-->Spark Project Scala
4)Tab till Build Tools-->Maven
5)Tab till Next button and press enter
6)Tab till tools-->Azure-->Azure Sign in and sign into Azure account and select any subscription
7)Tab till HDInsight and link a cluster
8)Navigate to project name-->src-->main-->scala and use keyboard shortcut key to open the context menu
9)Navigate to New-->Scala Class and press enter
10)Tab till object from menu and select it using keyboard
11)Override the code and Navigate to project name-->src-->main-->scala-->Project Name and right click on it & Tab till Submit Spark Application and press enter
12)Fill all the required details & Tab till finish button and press enter
13)Navigate to jobs under the cluster (which we have used) and press enter
14) Select any one the application and press enter. 
15)Verify if User can be able to access information present for jobs & Drivers with the help of keyboard

**Actual Result: ​** 
User cannot be able to access “Driver” and” Jobs” with the help of keyboard

**Expected Result: ​**
User should be able to access tooltip information and should be able to select ‘Jobs’ and details of individual Job information with the help of keyboard

**User Impact: ​**
Keyboard users will not be able to see the Job and Driver details of those are not accessible with the help of keyboard

**MAS Reference: ​**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

**Attachment:**
[Bug_3597_Jobs not accessible using keyboard.pptx](https://github.com/microsoft/azure-tools-for-java/files/3641834/Bug_3597_Jobs.not.accessible.using.keyboard.pptx)

**Comment by t-rufang** [comment]
Note: When focus is at `Application Graph`, we can't use tab or down button to reach the node like `Job 0` in the job graph. Therefore, we can't see the tooltip on the node.
![image](https://user-images.githubusercontent.com/32627233/65598241-91dd4d00-dfcd-11e9-802e-914a1d2c4b98.png)

One more thing is when we `click` the `Job 0` node, it shows the calculator `collect` in the node, however, we expect this `click` action should be achieved by keyboard.
![image](https://user-images.githubusercontent.com/32627233/65598458-0c0dd180-dfce-11e9-8eda-618538c53c36.png)

**Comment by t-rufang** [comment]
The job view page is wrote as static web pages with the help of JavaFX WebView which has no support for Accessibility. Please refer to JDK issue [JDK-8177382](https://bugs.openjdk.java.net/browse/JDK-8177382)

**Comment by Alexandria-R** [comment]
This has been approved to close by SME.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749700
